channels/ir.m3u

```diff
@@ -158,6 +158,6 @@ https://ir3.live21.ir/hls2/valiasr.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/7uEI6ck.png" group-title="News",VOA Persian
 https://voa-lh.akamaihd.net/i/voapnn_7@72817/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/v2wS68r.png" group-title="Religious",Nour TV
-https://184.75.213.27/liveevent.m3u8
+http://184.75.213.27/liveevent.m3u8
```

Comment:
I don't know exactly what the reason is, but this link doesn't work for me right now.
<img width="987" alt="Captura de pantalla 2020-01-13 a las 16 04 10" src="https://user-images.githubusercontent.com/7253922/72258200-836b7d80-361e-11ea-81c2-6e0f2c03af27.png">

Reply:
Yes, https://streamtest.in/ can't test this URL but playing on players.
Are you can play this URL with VLC?

Reply:
No, I can't. It return HTTP error 400.
<img width="735" alt="Captura de pantalla 2020-01-13 a las 17 02 21" src="https://user-images.githubusercontent.com/7253922/72261775-8c604d00-3626-11ea-853f-778cdf98b956.png">

Reply:
OK, I think it's working only in my country.
Please remove it.